src/index.html

```diff
@@ -18,7 +18,7 @@
       </div>
       <div class="nav-bar__home-search">
         <div class="search-bar">
```

Comment:
Should we change this div to a form element since it is holding a form input (even though it is a very short form)?

Reply:
I changed it. It doesn't change what the screen reader says, but I agree that it's better (semantically speaking)